### PR TITLE
Php8 compatibility

### LINKED ISSUE #16: PHP 8 Compatibility
Need to use this bundle on PHP8.0.

A PR will follow.

### PR DESCRIPTION
**Pull request by benoitStein**
Fixes #16

### CODE REVIEW

File: Validator/Constraints/EmbeddedAddressFormatConstraint.php [hunk 1]
```diff
@@ -10,13 +11,21 @@
...
 class EmbeddedAddressFormatConstraint extends AddressFormatConstraint
 {
-    /**
-     * {@inheritdoc}
-     */
-    public function getTargets()
-    {
-        return self::PROPERTY_CONSTRAINT;
-    }
+  public function __construct($options = null, $fieldOverrides = [])
```
Comment (daften) [inline Validator/Constraints/EmbeddedAddressFormatConstraint.php]: Please don't change the indentation like now. The PSR standards use 4 spaces, keep it as such.

File: Validator/Constraints/EmbeddedAddressFormatConstraint.php [hunk 2]
```diff
@@ -10,13 +11,21 @@
...
+  {
+    $this->fieldOverrides = new FieldOverrides($fieldOverrides);
```
Comment (daften) [inline Validator/Constraints/EmbeddedAddressFormatConstraint.php]: Why is this change here? This is not related to PHP8 compatibility. Please remove this. AFAICS the constructor in here is not necessary, since there are no local variables.

**Review by daften — CHANGES_REQUESTED** [review]
Please make sure this MR only contains changes relevant for PHP8 compatibility, so no whitespace changes or new constructors.

**Comment by benoitStein** [comment]
PR updated ! Just for php8 compatibility

**Comment by daften** [comment]
Merged, doesn't look like this could break anything for existing installs, since it's mostly the README file. 

@benoitStein  thank you for the contribution and let me know if you need a new version to be released!
